src/pepr/logger.ts

```diff
@@ -0,0 +1,15 @@
+import { Log } from "pepr";
+
+export function childLog(subproject: string) {
```

Comment:
It might be good to make this function named something like `setupLogger` or `getLogger`?

Reply:
agreed, i think setupLogger() makes more sense to me.